### PR TITLE
Register zakaria-didah.is-a.dev

### PR DESCRIPTION
**Pull request by zakaria-didah**
Register zakaria-didah.is-a.dev with A record pointing to 158.179.216.163.

### CODE REVIEW

File: domains/zakaria-didah.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "hghfth",
+           "email": "zakariadidah@gmail.com",
+           "discord": "788397929308225546"
+        },
+    
+        "record": {
+            "A": ["158.179.216.163"]
```
Comment (CuteDog5695) [inline domains/zakaria-didah.json:9]: cant visit site (it took too long to respond)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by triach-rold — CHANGES_REQUESTED** [review]
Unreachable.

File: domains/zakaria-didah.json (hunk 1 above)
Comment (zakaria-didah) [reply domains/zakaria-didah.json:9]: sorry. It's fully functional now.

Comment (zakaria-didah) [reply domains/zakaria-didah.json:9]: cuz we are not lunching the project today. but it's only a week so it's gonna be a nice idea to have your subdomain ready from now.

**Pull request closed without merging**